scripts/rollup/bundles.js

```diff
@@ -45,7 +45,12 @@ const bundles = [
       sourceMap: false,
     },
     entry: 'src/isomorphic/React.js',
-    externals: [],
+    externals: [
+      'create-react-class/factory',
+      'prop-types',
+      'prop-types/checkPropTypes',
```

Comment:
DOM does check too but it uses `React.checkPropTypes`. Rollup also only complained about the externals reference for isomorphic.